packages/web-pkg/src/components/AppTemplates/AppWrapper.vue

```diff
@@ -1,5 +1,7 @@
 <template>
   <main :id="applicationId" class="oc-height-1-1" @keydown.esc="closeApp">
+    <!-- TODO: Figure out what to write into hidden pageTitle for screenreader compatibility -->
+    <!-- <h1 class="oc-invisible-sr" v-text="pageTitle" /> -->
```

Comment:
Maybe we can retrieve the appName via the applicationId from the store for regular apps, the external app could emit update:applicationName as soon as it's available...